core/arch/arm/include/kernel/cache_helpers.h

```diff
@@ -43,6 +43,14 @@ static inline unsigned int dcache_get_line_size(void)
 	return CTR_WORD_SIZE <<
 		((value >> CTR_DMINLINE_SHIFT) & CTR_DMINLINE_MASK);
 }
+
+/*
+ * Get system outer cache line size.
+ */
+static inline unsigned int cache_get_line_size(void)
```

Comment:
How about calling this function `cache_get_max_line_size()` to align better with `CFG_MAX_CACHE_LINE_SHIFT`. By the way I think we can replace "outer line size" with "max line size" since that what we care about.

Reply:
OK.

Reply:
Updated the PR